tic_tac_toe/game.py

```diff
@@ -63,22 +73,69 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    winning_combinations = [
+        # Horizontal - top
+        ((0,0),(0,1),(0,2)),
+
+        # Horizontal - middle
+        ((1,0),(1,1),(1,2)),
+
+        # Horizontal - bottom
+        ((2,0),(2,1),(2,2)),
+
+        # Vertical - left
+        ((0,0),(1,0),(2,0)),
+
+        # Vertical - middle
+        ((0,1),(1,1),(2,1)),
+
+        # Vertical - right
+        ((0,2),(1,2),(2,2)),
+
+        # Diagonal - forward
+        ((0,2),(1,1),(2,0)),
+
+        # Diagonal - backward
+        ((0,0),(1,1),(2,2)),
+    ]
+
+    for combination in winning_combinations:
+        if _is_winning_combination(board, combination, player):
+            return player
+        
+    return None
 
 
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
+    
+    game = {
+    'player1': player1,
+    'player2': player2,
+    'board': [
+        ["-", "-", "-"],
+        ["-", "-", "-"],
+        ["-", "-", "-"],
+    ],
+    'next_turn': player1,
+    'winner': None
+    }
+    
+    return game
 
 
 def get_winner(game):
     """
     Returns the winner player if any, or None otherwise.
     """
-    pass
+    
+    if game['winner'] == None:
+        return None
+    else:
+        return game['winner']
 
```

Comment:
I know we are tasked with providing a get_winner function, but it seems like this function as written perhaps doesn't need to exist as it could be replaced with game['winner'], which would provide None, Player1, or Player2